### PR TITLE
Add PayPal instructions to README

### PR DESCRIPTION
**Pull request by intelliot**
- Steps for setting up PayPal with PopupBridge iOS
- Create `popupbridge-paypaldatacollector-ios.md`
  - Code snippets for data collector integration

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -201,6 +195,23 @@ PayPal authentication occurs in a popup window. However, this causes issues with
 
 PopupBridge solves this problem by allowing [`braintree-web`](https://github.com/braintree/braintree-web) to open the PayPal popup from a secure mini-browser.
 
+Using PayPal in a WebView
+-------------------------
+
+WebView-based checkout flows can accept PayPal with PopupBridge the Braintree JS SDK. For the authentication flow, PayPal requires a popup window—which can be simulated with PopupBridge.
```
Comment (skunkworks) [inline README.md]: `WebView-based checkout flows can accept PayPal with PopupBridge and the Braintree JS SDK.` and I might just omit the second sentence.

File: README.md [hunk 2]
```diff
@@ -201,6 +195,23 @@ PayPal authentication occurs in a popup window. However, this causes issues with
...
+
+### Setup
+1. Create a web-based checkout that accepts PayPal using Braintree JS v3.9.0 or higher
+2. Show a loading indicator
```
Comment (skunkworks) [inline README.md]: Might want to mention _when_ the loading indicator should be shown and hidden, e.g.

```
Show a loading indicator when the PayPal button is clicked
   - Hide the loading indicator in the PayPal tokenize completion callback
```

File: README.md [hunk 3]
```diff
@@ -201,6 +195,23 @@ PayPal authentication occurs in a popup window. However, this causes issues with
...
+    - PopupBridge lacks the PayPal loading page that customers see when using PayPal through a real popup window. We recommend adding a loading indicator on your web page when the PayPal button is clicked. In the PayPal tokenize completion callback, hide the loading indicator.
+3. Create a native mobile app that opens the checkout in a `WKWebView`
+4. Integrate the PopupBridge library
+5. Collect device data
+    - To help detect fraudulent activity, collect device data before performing PayPal transactions. This is similar to collecting device data with our [native iOS SDK](https://developers.braintreepayments.com/guides/paypal/vault/ios/v4) with a few differences:
+        1. Rather than the entire data collector, you can add just PayPalDataCollector to your app: `pod 'Braintree/PayPalDataCollector'`
```
Comment (skunkworks) [inline README.md]: `Rather than importing the entire data collector`

File: popupbridge-paypaldatacollector-ios.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+## One time payment
```
Comment (skunkworks) [inline popupbridge-paypaldatacollector-ios.md]: `One-time payments`

File: popupbridge-paypaldatacollector-ios.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+Implement the method that receives the popup URL and use this sample code to parse the `clientMetadataID`:
```
Comment (skunkworks) [inline popupbridge-paypaldatacollector-ios.md]: It's a bit ambiguous what the client metadata ID is, what merchants should do with the client metadata ID, what the `result` is, and how that relates to the `device data` that our merchants are already familiar with.

File: popupbridge-paypaldatacollector-ios.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+        }];
+    }
+}
+```
+3. On your web page, send a message to PopupBridge to request device data.
+```javascript
+window.popupBridge.sendMessage('requestDeviceData');
+```
+
+If you have questions, create an Issue for this GitHub project.
```
Comment (skunkworks) [inline popupbridge-paypaldatacollector-ios.md]: `+If you have questions, create an issue for this GitHub project.`

**Comment by intelliot** [comment]
@skunkworks Could you have another look at this? Thanks!

File: README.md [hunk 7]
```diff
@@ -36,7 +36,7 @@ Quick Start
   - Click **[+]** to add a new URL type
   - Under **URL Schemes**, enter a unique URL scheme, e.g. `com.my-app.popupbridge`
 
-2. In your application delegate, set up PopupBridge with the URL scheme:
+1. In your application delegate, set up PopupBridge with the URL scheme:
 
   ```objectivec
```
Comment (skunkworks) [inline README.md]: Code block indentation needs to be lined up one space over

**Pull request merged**